srcpkgs/nwipe/patches/0001-musl-build-fix.patch

```diff
@@ -0,0 +1,27 @@
+From 8871bf692e3b5012fdb0ea069b81c6a8db8ef85f Mon Sep 17 00:00:00 2001
+From: Nathan Owens <ndowens@artixlinux.org>
+Date: Mon, 21 Dec 2020 12:05:43 -0600
+Subject: [PATCH] musl build fix
+
+musl does not have bits/sigthread.h
+---
+ src/nwipe.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git src/nwipe.c src/nwipe.c
+index 2c9c851..487177e 100644
+--- src/nwipe.c
++++ src/nwipe.c
+@@ -32,7 +32,9 @@
+ #include <signal.h>
+ #include <pthread.h>
+ #include <sys/types.h>
++#if defined(__GLIBC__)
+ #include <bits/sigthread.h>
++#endif
+ 
```

Comment:
If you want to upstream this (please do :) ), just tell upstream that it's completely unnecessary. This is a glibc internal file, you are never supposed to touch it yourself.

They should just define something like `#define _XOPEN_SOURCE 500` at the top of their file, if it doesn't build now.

Reply:
Linked issue opened upstream